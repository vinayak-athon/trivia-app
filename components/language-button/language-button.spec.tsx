import React from "react";
import { render } from "@Test/utils";
import { LanguageButton } from "@Components";

describe("Components", () => {
  describe("LanguageButton with isActive=true", () => {
    const { container } = render(
      <LanguageButton lang="en" isActive onClick={() => {}} />
    );

    it("should match snapshot", () => {
      expect(container).toMatchSnapshot();
    });
  });
});
