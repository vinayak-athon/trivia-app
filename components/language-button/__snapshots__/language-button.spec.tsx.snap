// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components LanguageButton with isActive=true should match snapshot 1`] = `
.c0 {
  cursor: pointer;
  color: #000000;
  background: #f9da2e;
  border-radius: 5px;
  padding: 5px 10px;
  font-family: Arial;
  font-weight: 700;
  font-size: 25px;
  -webkit-letter-spacing: 0;
  -moz-letter-spacing: 0;
  -ms-letter-spacing: 0;
  letter-spacing: 0;
  text-transform: uppercase;
}

.c1 {
  background-color: green;
  color: inherit;
  font-size: 10px;
  border: none;
}

<div>
  <button
    class="c0 c1"
  >
    en
  </button>
</div>
`;
